### PR TITLE
[4.2] Actions Dropdown

### PR DESCRIPTION
**Pull request by brianteeman**
There were two _bugs_

1. The dropdown scripts was only being loaded on a split button - it was needed for both
2. Instead of using the htmlhelper it should have been using trhe web asset manager

To test disable the User Status Module and apply this pr

Go to any list view and check that the actions dropdown works as expected

Pull Request for Issue #38640

### CODE REVIEW

**Comment by Abernyte-Git** [comment]
I have tested this item :white_check_mark: successfully on fa0849e100906520c6f16766af774a6257ef0be2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38644">issues.joomla.org/tracker/joomla-cms/38644</a>.</sub>

**Comment by chmst** [comment]
Seems that there are more files in the PR as needed?

**Comment by brianteeman** [comment]
> Seems that there are more files in the PR than needed?

fixed

File: layouts/joomla/toolbar/dropdown.php [hunk 1]
```diff
@@ -35,12 +34,14 @@
 
 $direction = Factory::getLanguage()->isRtl() ? 'dropdown-menu-end' : '';
 
-?>
+/** @var Joomla\CMS\WebAsset\WebAssetManager $wa */
+$wa = Factory::getApplication()->getDocument()->getWebAssetManager();
+$wa->useScript('bootstrap.dropdown', '.dropdown');
```
Comment (dgrammatiko) [inline layouts/joomla/toolbar/dropdown.php]: This is wrong, there is only one param for the useScript. Also the web asset manager loads just the script, the HTMLHelper instantiates the script with the given params, in this case with the `.dropdown` class

Comment (Fedik) [reply layouts/joomla/toolbar/dropdown.php]: @brianteeman Dimitris is right. Just use `$wa->useScript('bootstrap.dropdown');` here, 
the js will handle default selectors and stuff.

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 16bcfe00002f2ad74c9cddeb31351fb654d9f08a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38644">issues.joomla.org/tracker/joomla-cms/38644</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 16bcfe00002f2ad74c9cddeb31351fb654d9f08a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38644">issues.joomla.org/tracker/joomla-cms/38644</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38644">issues.joomla.org/tracker/joomla-cms/38644</a>.</sub>

**Comment by wilsonge** [comment]
OK This one wasn't intuitive to me - had to dig back through the comments on the previous issue - added a code comment with reasoning to help stop it being reverted accidentally in the future. RTC is still valid

**Comment by brianteeman** [comment]
thanks

**Pull request merged**